config/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "appid": "",
   "group": "",
-  "webscene": "19faa71a3bf6468cae35b4fce9393a7d",
+  "webscene": "",
```

Comment:
But that's much more an "ok-that-makes-sense" behavior than to supply a group id and get a web scene.